docs/source/about.rst

```diff
@@ -0,0 +1,297 @@
+.. _about:
+
+About
+=====
+
+What is Surround exactly?
+*************************
+
+Surround is an open-source framework developed by the Applied Artificial Intelligence Institute (`A`:superscript:`2`\ `I`:superscript:`2`) to take machine
```

Comment:
Can you please add the link to the Applied Artificial Intelligence Institute's website here?